samples/app-anonymous-users/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-anonymous-users-csharp",
+    "source": "officeDev",
+    "title": "Anonymous User Support",
+    "shortDescription": "This sample shows Anonymous Users Support In Meeting.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-anonymous-users/csharp",
+    "longDescription": [
+      "This sample shows Anonymous Users Support In Meeting."
+    ],
+    "creationDateTime": "2023-02-06",
+    "updateDateTime": "2023-02-08",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/app-anonymous-users/csharp/Images/appanonymoususersGif.gif",
+        "alt": "Solution UX showing Anonymous User Support"
```

Comment:
Solution UX showing anonymous user support in teams meeting apps

Reply:
Fixed